test/unity/toolkits/drawing_classifier/test_dc_serialization.cxx

```diff
@@ -21,9 +21,18 @@ namespace {
 class drawing_classifier_mock
     : public turi::drawing_classifier::drawing_classifier {
  public:
-  std::unique_ptr<neural_net::model_spec> my_init_model() {
+  std::unique_ptr<neural_net::model_spec> get_model_spec() const {
     return init_model();
   }
+
+  drawing_classifier_mock() = default;
+
+  drawing_classifier_mock(std::unique_ptr<neural_net::model_spec> ms)
+      : drawing_classifier({}, std::move(ms), nullptr, nullptr, nullptr) {}
```

Comment:
Let's have constructors precede other methods in the class definition

https://google.github.io/styleguide/cppguide.html#Declaration_Order